src/ProjectTemplates/BlazorTemplates.Tests/BlazorServerTemplateTest.cs

```diff
@@ -8,6 +8,7 @@
 using System.Runtime.InteropServices;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.BrowserTesting;
+using Microsoft.AspNetCore.Testing;
```

Comment:
Why❔

Reply:
I had some skipped/conditional earlier, might as well leave the include in, makes the likely future quarantining easier